### PR TITLE
Add Panko Event service

### PR DESCRIPTION
**Pull request by aufi**
Events loading from Ceilometer service will be moved to separated service
called Panko from Ocata Openstack release. Panko has own entry in Openstack
endpoint list with an "Event" service type.

Adding support for this service extracting its code from Metering.

https://github.com/openstack/panko

### CODE REVIEW

**Comment by aufi** [comment]
Example output:
```
$ be ruby examples/event/basics.rb 
### SERVICE CONNECTION ###
#<Fog::Event::OpenStack::Real:46926437349000 @openstack_auth_url="http://192.168.24.1:5000/v2.0/tokens" @openstack_username="admin" ...>
### LIST EVENTS ###
  <Fog::Event::OpenStack::Events
    [
                  <Fog::Event::OpenStack::Event
        message_id="aa270714-32b9-41ca-9efc-175e5ffffe88",
        event_type="network.create.start",
        generated="2017-02-16T14:59:07.912075",
        raw={},
        traits=[{"type"=>"string", "name"=>"project_id", "value"=>"d37176ad75cb463490a1fe86bc52764e"}, {"type"=>"string", "name"=>"request_id", "value"=>"req-6812ebda-05f1-45c7-a9d2-e24544466a03"}, {"type"=>"string", "name"=>"service", "value"=>"network.undercloud-0.redhat.local"}, {"type"=>"string", "name"=>"tenant_id", "value"=>"d37176ad75cb463490a1fe86bc52764e"}, {"type"=>"string", "name"=>"user_id", "value"=>"a45a5db5ca494f12aa78ed9847f6b26b"}]
      >,...  
    ]
  >
```

**Comment by aufi** [comment]
@seanhandley @Ladas Hi, may I ask for a review. This PRs is similar to https://github.com/fog/fog-openstack/pull/224, but it is a different service (event) separated from Ceilometer (metering) service.

File: lib/fog/event/openstack.rb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+           :openstack_auth_token     => @auth_token,
+           :openstack_management_url => @openstack_management_url}
+        end
+      end
+
+      class Real
+        include Fog::OpenStack::Core
+
+        def self.not_found_class
+          Fog::Metering::OpenStack::NotFound
```
Comment (Carthaca) [inline lib/fog/event/openstack.rb]: I suggest `Fog::Event::OpenStack::NotFound` here

File: lib/fog/openstack.rb [hunk 2]
```diff
@@ -59,6 +59,10 @@ module Metric
     autoload :OpenStack, File.expand_path('../metric/openstack', __FILE__)
   end
 
+  module Event
```
Comment (Carthaca) [inline lib/fog/openstack.rb]: Please put it in alphabetical order ;)

File: test/requests/event/event_tests.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+require "test_helper"
+
+describe "Fog::Event[:openstack] | event requests" do
+  before do
+    @metering = Fog::Metering[:openstack]
```
Comment (Carthaca) [inline test/requests/event/event_tests.rb]: This should test Fog::Event

**Comment by Ladas** [comment]
@aufi great addition, thank you :+1:

File: lib/fog/event/openstack.rb (hunk 1 above)
Comment (aufi) [inline lib/fog/event/openstack.rb]: Fixed, thanks.

File: lib/fog/openstack.rb (hunk 2 above)
Comment (aufi) [inline lib/fog/openstack.rb]: Yep, created a new commit (part of this PR) updating also some other service names.

File: test/requests/event/event_tests.rb (hunk 3 above)
Comment (aufi) [inline test/requests/event/event_tests.rb]: Fixed, thanks.

**Comment by Carthaca** [comment]
Thanks!

**Comment by tzumainn** [comment]
@miq-bot add_label euwe/yes

**Pull request merged**